src/serial-provision-dialog.ts

```diff
@@ -54,101 +64,115 @@ class SerialProvisionDialog extends LitElement {
 
   @state() private _hasProvisioned = false;
 
-  @state() private _showProvisionForm = false;
```

Comment:
No more "Dashboard" and "Provision" form. it's just one step now and the "dashboard" is only for the provisioned state now.

Reply:
I saw in the screenshot that the provisioned state still has device info above it, I think that we can drop it at that point?